src/pages/login/login.component.tsx

```diff
@@ -0,0 +1,34 @@
+import UserForm from "../../components/userForm/userForm.component"
+import {Link} from "react-router-dom"
+
+import "./login.styles.css"
+
+function LoginPage() {
+	console.log("LOGIN PAGE RENDERED")
```

Comment:
Do not push console statements to remote repo.

Reply:
Removed.